src/main.tf

```diff
@@ -0,0 +1,61 @@
+locals {
+  create_accepter = var.accepter_vnet_id != null ? true : false
```

Comment:
First, this doesn't need to be a ternary. Using a ternary to set a boolean just makes it harder to read. This can just be `create_accepter = var.accepter_vnet_id != null`.

Second, I think the logic needs to be reversed here, and therefore anywhere its referenced. If a user specifies a accepter VNet ID in the params, that implies that **DON'T** want to manage the accepter side because its not a Massdriver VNet. They will need to manage it.  The connection is when we **DO** manage it, the param is when we **DON'T**. This logic says that if the param isn't null, then we do manage it.

Realistically the connection should take precedence. If the accepter connection is specified then that should take precedence and we should assume that we are managing the accepter. If it isn't specified, then we assume they passed us the param, and we don't manage it.